integration_tests/tests/consistency_pardot__campaigns.sql

```diff
@@ -3,16 +3,25 @@
     enabled=var('fivetran_validation_tests_enabled', false)
 ) }}
 
-{% set exclude_cols = var('consistency_test_exclude_metrics', []) %}
+{% set exclude_cols = var('consistency_test_exclude_metrics', ['sum_opportunity_amount_lost', 'sum_opportunity_amount_won', 'sum_opportunity_amount_open']) %}
```

Comment:
So you don't have to re-add the `sum_*`s to the var if you need to add more cols.
```suggestion
{% set exclude_cols = var('consistency_test_exclude_metrics', []) + ['sum_opportunity_amount_lost', 'sum_opportunity_amount_won', 'sum_opportunity_amount_open']) %}
```